src/app/space/create/codebases/codebases-delete/codebase-delete-dialog.component.html

```diff
@@ -1,6 +1,14 @@
-<span>You are about to remove codebase <b>{{codebase.name}}</b> from your space.
-    Please, verify this action by clicking Remove below.</span>
-<div class="remove-footer pull-right">
-  <button class="btn btn-default" (click)="cancel()">Cancel</button>
-  <button class="btn btn-danger" (click)="confirmDelete()">Remove</button>
+<div class="modal-header">
+  <h4 class="modal-title pull-left">Remove Codebase</h4>
+  <button type="button" class="close pull-right" aria-label="Close" (click)="host.hide()">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+<div class="modal-body">
+    <span>You are about to remove codebase <b>{{codebase.name}}</b> from your space.
+      Please, verify this action by clicking Remove below.</span>
+</div>
+<div class="modal-footer">
+    <button class="btn btn-default" (click)="cancel()">Cancel</button>
+    <button class="btn btn-danger" (click)="confirmDelete()">Remove</button>
```

Comment:
Not sure what's causing it, but this dialog gains some "OBJ" artifacts in its bottom left corner with this applied.

Before:
![screenshot from 2018-02-08 13-47-38](https://user-images.githubusercontent.com/3787464/35991918-b6332902-0cd6-11e8-932e-61e0a7466a90.png)

After:
![screenshot from 2018-02-08 13-48-33](https://user-images.githubusercontent.com/3787464/35991941-c90feefc-0cd6-11e8-920b-598e11d48b1c.png)

Reply:
This occurs locally ? My local copy doesn't have this issue ..
<img width="1440" alt="screen shot 2018-02-08 at 3 35 39 pm" src="https://user-images.githubusercontent.com/17230733/35998409-7d65c026-0cea-11e8-937e-2e4516d6cc8d.png">